Tools/buildbot/test.bat

```diff
@@ -37,7 +37,12 @@ set TEMP_ARGS=--temp %REMOTE_PYTHON_DIR%temp
 
 set rt_args=%rt_opts% %dashU% -rwW --slowest --timeout=1200 --fail-env-changed %regrtest_args% %TEMP_ARGS%
 ssh %SSH_SERVER% "set TEMP=%REMOTE_PYTHON_DIR%temp& %REMOTE_PYTHON_DIR%PCbuild\rt.bat" %rt_args%
```

Comment:
Maybe change this to 
ssh %SSH_SERVER% "set TEMP=%REMOTE_PYTHON_DIR%temp & cd %REMOTE_PYTHON_DIR% & PCbuild\rt.bat" %rt_args%